### PR TITLE
Add Copyright licenses that can't be directly scraped

### PR DESCRIPTION
**Pull request by prayutsu**
Added the licenses that can't be scraped directly.
1. http://www.opensource.org/licenses/bsd-license
2. https://github.com/javaee/javax.annotation/blob/master/LICENSE
3. https://github.com/DanielMartinus/Konfetti/blob/main/LICENSE
4. http://opensource.org/licenses/MIT
5. https://asm.ow2.io/license.html

### CODE REVIEW

File: bsd-3-clause-license.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ASM: a very small and fast Java bytecode manipulation framework
```
Comment (BenHenning) [inline bsd-3-clause-license.txt]: This seems specific to ASM--should this file be named for the specific dependency that needs it, instead?

File: cddl-gpl2-license.txt [hunk 2]
```diff
@@ -0,0 +1,759 @@
+COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.1
```
Comment (BenHenning) [inline cddl-gpl2-license.txt]: Which dependencies require these?

File: konfetti-license.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ISC License
```
Comment (BenHenning) [inline konfetti-license.txt]: Re: name, I suggest including the whole Maven coordinate minus version to reduce potential ambiguities in the future between different license files.

File: simplified-bds-license.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Note: This license has also been called the "Simplified BSD License" and the "FreeBSD License". See also the 3-clause BSD License.
```
Comment (BenHenning) [inline simplified-bds-license.txt]: Typo in filename: s/bds/bsd

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @prayutsu. Leaving some follow-up comments.

File: cddl-gpl2-license.txt (hunk 2 above)
Comment (prayutsu) [reply cddl-gpl2-license.txt]: `javax.annotation:javax.annotation-api:1.3.2`

The above dependency requires this.

File: simplified-bds-license.txt (hunk 4 above)
Comment (prayutsu) [reply simplified-bds-license.txt]: Fixed.

File: konfetti-license.txt (hunk 3 above)
Comment (prayutsu) [reply konfetti-license.txt]: Done.

File: bsd-3-clause-license.txt (hunk 1 above)
Comment (prayutsu) [reply bsd-3-clause-license.txt]: Yup, sounds good.

File: cddl-gpl2-license.txt (hunk 2 above)
Comment (prayutsu) [reply cddl-gpl2-license.txt]: As only one dependency requires this license, updated the name of the file to be specific to the dependency.

File: org.ow2.asm.asm-7.2-license.txt [hunk 5]
```diff
@@ -0,0 +1,27 @@
+ASM: a very small and fast Java bytecode manipulation framework
```
Comment (BenHenning) [inline org.ow2.asm.asm-7.2-license.txt]: We should make sure the filenames are consistent. It seems like the javax one is using the full coordinate, but konfetti and asm are using part.

**Review by BenHenning — APPROVED** [review]
Thanks @prayutsu.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Actually requesting changes per the last unresolved comment.

File: org.ow2.asm.asm-7.2-license.txt (hunk 5 above)
Comment (prayutsu) [reply org.ow2.asm.asm-7.2-license.txt]: Fixed.

**Comment by prayutsu** [comment]
@BenHenning PTAL fixed the naming of the 2 licenses.

**Review by BenHenning — APPROVED** [review]
Thanks @prayutsu.

**Pull request merged**